src/Configuration/ValueObject/DatabaseConfig.php

```diff
@@ -29,7 +29,7 @@ public static function fromArray(array $data): self
 
         return new self(
             $data['host'],
-            $data['port'] ?? null,
+            $data['port'] ? (string) $data['port'] : null,
```

Comment:
radši jsem to tu přidal (kdyby ještě přišel odněkud int)

Reply:
ok